### ISSUE TITLE
Limits don't show count in nether island or end island

### ISSUE THREAD

**Issue by LordSheep** [BentoBoxWorld/Limits#43]
**Description**
When I place a mob like a chicken in the overworld island it counts it (ex: 1/100) but in the nether and end it always shows "0/100" chickens even if there are chickens.

**Steps to reproduce the behavior:**
1. Install latest limits addon (Limits-1.6.1-SNAPSHOT-b164.jar)
2. Start server to load folder and configure limits in config.yml.
3. Start server and place an entity in nether or end.
4. Do /is limits and look at count.

**Expected behavior**
Showing exact number of specific entities on island.

**Screenshots**
![image](https://user-images.githubusercontent.com/30538383/64279040-2c40f800-cf1c-11e9-9c48-4aef171984cd.png)

**Server Information:**

[Please complete the following information:]
 - Database being used (YAML, JSON, MySQL, MongoDB): [] JSON
 - OS: [e.g. iOS] Don't know
 - Java Version: [e.g. Java 8] Don't know
 - BentoBox version: [e.g. 1.7.2.21] 1.6.0
 - Addons installed? [Do '/bentobox version' and copy/paste from the console] (doesn't show stuff in console and too lazy to type up so I just screenshotted)
![image](https://user-images.githubusercontent.com/30538383/64279250-99ed2400-cf1c-11e9-8491-63c604efc41b.png)

 - Other plugins? [Do '/plugins' and copy/paste from the console]
![image](https://user-images.githubusercontent.com/30538383/64279195-7aee9200-cf1c-11e9-8fec-d0fc78a9e26e.png)

**Additional context**
It still limits spawning correctly.
![image](https://user-images.githubusercontent.com/30538383/64279421-00724200-cf1d-11e9-8325-f44292da2552.png)

**Comment by SirvydasS** [comment]
Overworld and Nether animals limits is different, if limit is 10 pigs, you can have 10 in nether and 10 in overworld BUT if hopper limit is 20 you can have through all world only 20 not more. There is a mistake somewhere. Using now Limits 1.9.0

**Comment by tastybento** [comment]
I added checking for entities in nether and end. Please try 1.9.1 snapshot build.

**Comment by SirvydasS** [comment]
Some still interesting bug. If I were in the overworld and no one in nether the entity doesn't add up in /is limits, but if someone else comes to nether then it counts and shows.

**Comment by tastybento** [comment]
Ahhh. Hmm. This is because entities are not loaded if no one is in the area. This could technically also occur in the overworld if you had a huge island. There is not a good way to fix this without loading chunks, which is not a good idea. Hmmm. Not sure what to do here.

**Comment by SirvydasS** [comment]
If you don't come up with anything, maybe make all limits different in each word.

**Comment by tastybento** [comment]
Yes, I think that we'll have to do that. All three "worlds" will have to be separated.

**Comment by SirvydasS** [comment]
Can't wait :)
